### PR TITLE
chore: add several metrics for dwh

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: pkg/eventpersisterdwh/persister/evaluation_event.go [hunk 1]
```diff
@@ -87,6 +87,8 @@ func (w *evalEvtWriter) Write(
 			case *eventproto.EvaluationEvent:
 				e, retriable, err := w.convToEvaluationEvent(ctx, evt, id, environmentNamespace)
 				if err != nil {
+					// If there is nothing to link, we don't report it as an error
+					handledCounter.WithLabelValues(codeNoLink).Inc()
```
Comment (cre8ivejp) [inline pkg/eventpersisterdwh/persister/evaluation_event.go:91]: Can you also add `Linked` metrics?

E.g.
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersisterops/persister/evaluation_event.go#L90
https://github.com/bucketeer-io/bucketeer/blob/main/pkg/eventpersisterops/persister/goal_event.go#L97

**Review by cre8ivejp — CHANGES_REQUESTED** [review]

**Comment by ono-max** [comment]
@cre8ivejp 

Thank you. I've updated.

**Review by masaaania — APPROVED** [review]
LGTM!

**Review by cre8ivejp — APPROVED** [review]
Good good!

**Pull request merged**